### PR TITLE
[DX-695] Updating steps for redhat-rhel-centos.md

### PR DESCRIPTION
**Pull request by carlosvillanua**
### For internal users - please add a Jira DX PR ticket to the subject!
---
### Preview Link

[preview](https://deploy-preview-3261--tyk-docs.netlify.app/docs/nightly/tyk-on-premises/redhat-rhel-centos)

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md [hunk 1]
```diff
@@ -11,13 +11,112 @@ aliases:
   - /tyk-api-gateway-v-2-0/installation-options-setup/install-tyk-pro-edition-on-red-hat/
   - /getting-started/installation/with-tyk-on-premises/redhat-rhel-centos/
 ---
+
+Select the prefer way of installing Tyk by selecting **Shell** or **Ansible** tab for instructions.
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md]:
```suggestion

Select the preferred way of installing Tyk by selecting **Shell** or **Ansible** tab for instructions.
```

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md [hunk 2]
```diff
@@ -11,13 +11,112 @@ aliases:
...
+| CentOS | 7 | ✅ |
+| RHEL | 8 | ✅ |
+| RHEL | 7 | ✅ |
+
+
+## Getting Started
+
+### Installing Redis
+
+Tyk Gateway has a [dependency](https://tyk.io/docs/planning-for-production/redis/#supported-versions) on Redis. Follow the steps provided by Red Hat to make the installation of Redis, conducting a [search](https://access.redhat.com/search/?q=redis) for the correct version and distribution.
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md]: Use relative links, e.g. {{< ref "/planning-for-production/redis#supported-versions" >}}

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md [hunk 3]
```diff
@@ -11,13 +11,112 @@ aliases:
...
+
+### Installing PostgreSQL or MongoDB
+
+Tyk Dashboard has a dependency on a storage database that can be [PostgreSQL](https://tyk.io/docs/planning-for-production/database-settings/postgresql/) or [MongoDB](https://tyk.io/docs/planning-for-production/database-settings/mongodb/).
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md]: Use relative links, e.g. {{< ref "/planning-for-production/database-settings/postgresql" >}}

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md [hunk 4]
```diff
@@ -11,13 +11,112 @@ aliases:
...
+  
+{{< tabs_start >}}
+{{< tab_start "PostgreSQL" >}}
+### Installing PostgreSQL
+
+Check the PostgreSQL supported [versions](https://tyk.io/docs/planning-for-production/database-settings/postgresql/). Follow the steps provided by [PostgreSQL](https://www.postgresql.org/download/linux/redhat/) to install it.
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md]: Use relative links, e.g. {{< ref "/planning-for-production/database-settings/postgresql" >}}

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md [hunk 5]
```diff
@@ -11,13 +11,112 @@ aliases:
...
+ALTER ROLE tyk with PASSWORD '123456';
+```
+Create a DB (my example is tyk_analytics)
+```console
+sudo -u tyk createdb tyk_analytics
+```
+{{< tab_end >}}
+{{< tab_start "MongoDB" >}}
+<br>
+Check the MongoDB supported [versions](https://tyk.io/docs/planning-for-production/database-settings/postgresql/). Follow the steps provided by [MongoDB](https://www.mongodb.com/docs/manual/tutorial/install-mongodb-on-red-hat/) to install it.
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md]: Use relative links, e.g. {{< ref "/planning-for-production/database-settings/postgresql" >}}

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md [hunk 6]
```diff
@@ -11,13 +11,112 @@ aliases:
...
+The order is to install Tyk Dashboard, then Tyk Pump and then Tyk Gateway for a full stack.
+
+- [Dashboard]({{< ref "tyk-on-prem/installation/redhat-rhel-centos/dashboard" >}})
+- [Pump]({{< ref "tyk-on-prem/installation/redhat-rhel-centos/analytics-pump" >}})
+- [Gateway]({{< ref "tyk-on-prem/installation/redhat-rhel-centos/gateway" >}})
+
+{{< note success >}}
+**Note**  
+
+For a production environment, we recommend that the Tyk Gateway, Tyk Dashboard and Tyk Pump are installed on separate machines. If installing multiple Tyk Gateways, you should install each on a separate machine. See [Planning for Production]({{< ref "planning-for-production" >}}) For more details.
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md]:
```suggestion
For a production environment, we recommend that the Tyk Gateway, Tyk Dashboard and Tyk Pump are installed on separate machines. If installing multiple Tyk Gateways, you should install each on a separate machine. See [Planning for Production]({{< ref "planning-for-production" >}}) for more details.
```

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md [hunk 7]
```diff
@@ -11,13 +11,112 @@ aliases:
   - /tyk-api-gateway-v-2-0/installation-options-setup/install-tyk-pro-edition-on-red-hat/
   - /getting-started/installation/with-tyk-on-premises/redhat-rhel-centos/
 ---
+
+Select the prefer way of installing Tyk by selecting **Shell** or **Ansible** tab for instructions.
+
 {{< tabs_start >}}
+{{< tab_start "Shell" >}}
+
+## Supported Distributions
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md:21]: If the CentOS and RHEL versions are the same for shell and Ansible, maybe include one supported distributions table before the tabs sentence (_Select the preferred way of installing Tyk by selecting Shell or Ansible tab for instructions_)

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md [hunk 8]
```diff
@@ -11,13 +11,112 @@ aliases:
...
+sudo systemctl enable mongod
+# start MongoDB server
+sudo systemctl start mongod
+```
+{{< tab_end >}}
+{{< tabs_end >}}
+
+## Install Tyk Self-Managed on Red Hat (RHEL) / CentOS
+
+You can install Tyk on RHEL or CentOS using our YUM repositories, follow the guides and tutorials in this section to have Tyk up and running in no time.
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md]:
```suggestion
You can install Tyk on RHEL or CentOS using our YUM repositories. Follow the guides and tutorials in this section to have Tyk up and running in no time.
```

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md [hunk 9]
```diff
@@ -11,13 +11,112 @@ aliases:
...
+## Supported Distributions
+| Distribution | Version | Supported |
+| --------- | :---------: | :---------: |
+| CentOS | 8 | ✅ |
+| CentOS | 7 | ✅ |
+| RHEL | 8 | ✅ |
+| RHEL | 7 | ✅ |
+
+
+## Getting Started
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md]: Is the content for the _shell_ tab split into 2 sections, i.e.

- Install and Configure Dependencies (currently the section named Getting Started)
- Install Tyk Self-Managed on Red Hat (RHEL) / CentOS?

If so then maybe rename _Getting Started_ to 

```Suggestion
## Install and Configure Dependencies
```

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md [hunk 10]
```diff
@@ -11,13 +11,112 @@ aliases:
...
+
+### Installing Redis
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md]:
```suggestion
### Redis
```

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md [hunk 11]
```diff
@@ -11,13 +11,112 @@ aliases:
...
+
+Tyk Gateway has a [dependency](https://tyk.io/docs/planning-for-production/redis/#supported-versions) on Redis. Follow the steps provided by Red Hat to make the installation of Redis, conducting a [search](https://access.redhat.com/search/?q=redis) for the correct version and distribution.
+
+### Installing PostgreSQL or MongoDB
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md]:
```suggestion
### Storage Database
```

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md [hunk 12]
```diff
@@ -11,13 +11,112 @@ aliases:
...
+
+Tyk Dashboard has a dependency on a storage database that can be [PostgreSQL](https://tyk.io/docs/planning-for-production/database-settings/postgresql/) or [MongoDB](https://tyk.io/docs/planning-for-production/database-settings/mongodb/).
+  
+{{< tabs_start >}}
+{{< tab_start "PostgreSQL" >}}
+### Installing PostgreSQL
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md]:
```suggestion
### Install PostgreSQL
```

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md [hunk 13]
```diff
@@ -11,13 +11,112 @@ aliases:
...
+
+Check the PostgreSQL supported [versions](https://tyk.io/docs/planning-for-production/database-settings/postgresql/). Follow the steps provided by [PostgreSQL](https://www.postgresql.org/download/linux/redhat/) to install it.
+
+### Configuring PostgreSQL
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md]:
```suggestion
Configure PostgreSQL
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Hi Carlos, PR is looking good to me!!!

Made some very minor suggestions, relating to structure and updating tyk.io/docs links to use *ref* links.

Once changes addressed and a peer review has been approved will merge and release

File: tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md (hunk 5 above)
Comment (carlosvillanua) [reply tyk-docs/content/tyk-on-premises/redhat-rhel-centos.md]: Done, Thank you!

**Review by dcs3spp — APPROVED** [review]

**Comment by dcs3spp** [comment]
@carlostyk @zalbiraw See that links are fixed. I've approved. Is this good to merge and release?

**Review by zalbiraw — APPROVED** [review]
LGTM

**Pull request merged**